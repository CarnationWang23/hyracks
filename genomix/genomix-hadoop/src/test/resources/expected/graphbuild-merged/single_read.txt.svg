<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.26.3 (20100126.1600)
 -->
<!-- Title: single_read_txt Pages: 1 -->
<svg width="206pt" height="321pt"
 viewBox="0.00 0.00 206.00 321.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph1" class="graph" transform="scale(1 1) rotate(0) translate(4 317)">
<title>single_read_txt</title>
<polygon fill="white" stroke="white" points="-4,5 -4,-317 203,-317 203,5 -4,5"/>
<g id="graph2" class="cluster"><title>cluster_legend</title>
<polygon fill="none" stroke="black" points="59,-8 59,-209 190,-209 190,-8 59,-8"/>
<text text-anchor="middle" x="124.5" y="-192.4" font-family="Times Roman,serif" font-size="14.00">legend</text>
</g>
<g id="graph3" class="cluster"><title>cluster_1</title>
<polygon fill="none" stroke="black" points="8,-217 8,-305 130,-305 130,-217 8,-217"/>
</g>
<!-- legend_0_0 -->
<g id="node2" class="node"><title>legend_0_0</title>
<ellipse fill="black" stroke="black" cx="69" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_0 -->
<g id="node3" class="node"><title>legend_1_0</title>
<ellipse fill="black" stroke="black" cx="180" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_0&#45;&gt;legend_1_0 -->
<g id="edge3" class="edge"><title>legend_0_0&#45;&gt;legend_1_0</title>
<path fill="none" stroke="#dd1e2f" d="M70.9266,-157C82.3997,-157 142.097,-157 167.994,-157"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="168.02,-160.5 178.02,-157 168.02,-153.5 168.02,-160.5"/>
<text text-anchor="middle" x="150" y="-162.4" font-family="Times Roman,serif" font-size="14.00">FF</text>
</g>
<!-- legend_0_1 -->
<g id="node5" class="node"><title>legend_0_1</title>
<ellipse fill="black" stroke="black" cx="69" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_1 -->
<g id="node6" class="node"><title>legend_1_1</title>
<ellipse fill="black" stroke="black" cx="180" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_1&#45;&gt;legend_1_1 -->
<g id="edge5" class="edge"><title>legend_0_1&#45;&gt;legend_1_1</title>
<path fill="none" stroke="#ebb035" d="M70.9266,-116C82.3997,-116 142.097,-116 167.994,-116"/>
<polygon fill="#ebb035" stroke="#ebb035" points="168.02,-119.5 178.02,-116 168.02,-112.5 168.02,-119.5"/>
<text text-anchor="middle" x="150" y="-121.4" font-family="Times Roman,serif" font-size="14.00">FR</text>
</g>
<!-- legend_0_2 -->
<g id="node8" class="node"><title>legend_0_2</title>
<ellipse fill="black" stroke="black" cx="69" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_2 -->
<g id="node9" class="node"><title>legend_1_2</title>
<ellipse fill="black" stroke="black" cx="180" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_2&#45;&gt;legend_1_2 -->
<g id="edge7" class="edge"><title>legend_0_2&#45;&gt;legend_1_2</title>
<path fill="none" stroke="#06a2cb" d="M70.9266,-75C82.3997,-75 142.097,-75 167.994,-75"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="168.02,-78.5001 178.02,-75 168.02,-71.5001 168.02,-78.5001"/>
<text text-anchor="middle" x="150" y="-80.4" font-family="Times Roman,serif" font-size="14.00">RF</text>
</g>
<!-- legend_0_3 -->
<g id="node11" class="node"><title>legend_0_3</title>
<ellipse fill="black" stroke="black" cx="69" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_3 -->
<g id="node12" class="node"><title>legend_1_3</title>
<ellipse fill="black" stroke="black" cx="180" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_3&#45;&gt;legend_1_3 -->
<g id="edge9" class="edge"><title>legend_0_3&#45;&gt;legend_1_3</title>
<path fill="none" stroke="#218559" d="M70.9266,-34C82.3997,-34 142.097,-34 167.994,-34"/>
<polygon fill="#218559" stroke="#218559" points="168.02,-37.5001 178.02,-34 168.02,-30.5001 168.02,-37.5001"/>
<text text-anchor="middle" x="150" y="-39.4" font-family="Times Roman,serif" font-size="14.00">RR</text>
</g>
<!-- 1,1 -->
<g id="node15" class="node"><title>1,1</title>
<ellipse fill="none" stroke="black" cx="69" cy="-261" rx="51.8276" ry="36.0624"/>
<text text-anchor="start" x="49" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="40,-253 40,-267 98,-267 98,-253 40,-253"/>
<text text-anchor="start" x="42.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">AATAGAAG</text>
<polygon fill="#218559" stroke="#218559" points="40,-239 40,-253 98,-253 98,-239 40,-239"/>
<text text-anchor="start" x="46.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">CTTCTATT</text>
</g>
</g>
</svg>
