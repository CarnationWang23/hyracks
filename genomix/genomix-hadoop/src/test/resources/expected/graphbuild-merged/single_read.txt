((1,1)	[]	[]	[]	[]	AATAGAAG)	(null)
